types/foundry-vtt-api/types/framework/entities/chatmessage.d.ts

```diff
@@ -0,0 +1,194 @@
+declare class Messages extends Collection<ChatMessage> {
+    entities: ChatMessage[];
+
+    /**
+     * Elements of the Messages collection are instances of the ChatMessage class
+     */
+    get object(): ChatMessage;
+
+    values(): IterableIterator<ChatMessage>;
+
+    /* -------------------------------------------- */
+    /*  Socket Listeners and Handlers
+	/* -------------------------------------------- */
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entities/chatmessage.d.ts;
  Line: 13;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.